### PR TITLE
ADCM-645 rewrite adcm check logic to serialize all checks of one job via DB

### PR DESCRIPTION
**Pull request by grey-kristy**

### CODE REVIEW

File: cm/job.py [hunk 1]
```diff
@@ -729,20 +731,22 @@ def log_check(job_id, title, res, msg):
...
         err('JOB_NOT_FOUND', f'no job with id #{job_id}')
+    cl = CheckLog(job_id=job.id, title=title, message=msg, result=res)
+    cl.save()
+    return cl
 
+
+def finish_check(job_id):
+    data = []
+    for cl in CheckLog.objects.filter(job_id=int(job_id)):
+        data.append({'title': cl.title, 'message': cl.message, 'result': cl.result})
```
Comment (a-alferov) [inline cm/job.py]: We can use the inline method values() for QuerySet object.
```
data = list(CheckLog.objects.filter(job_id=int(job_id)).values('title', 'message', 'result'))
```

File: cm/job.py [hunk 2]
```diff
@@ -729,20 +731,22 @@ def log_check(job_id, title, res, msg):
             err('JOB_NOT_FOUND', f'job #{job.id} has status "{job.status}", not "running"')
     except JobLog.DoesNotExist:
         err('JOB_NOT_FOUND', f'no job with id #{job_id}')
+    cl = CheckLog(job_id=job.id, title=title, message=msg, result=res)
+    cl.save()
+    return cl
```
Comment (a-alferov) [inline cm/job.py]: Is this `return cl` necessary here?

File: cm/job.py [hunk 3]
```diff
@@ -729,20 +731,22 @@ def log_check(job_id, title, res, msg):
...
-        raw = f.read()
-        data = json.loads(raw)
-        f.seek(0)
-    else:
-        f = open(log_name, 'w+')
-        data = []
-    data.append({'title': title, 'message': msg, 'result': res})
+    f = open(log_name, 'w+')
     f.write(json.dumps(data, indent=3))
     f.close()
```
[the remark this answers is not in the document]
Comment (grey-kristy) [reply cm/job.py]: ok

File: cm/job.py (hunk 2 above)
Comment (grey-kristy) [reply cm/job.py]: It is not used now, but can be used in future. Just in case

File: cm/job.py (hunk 1 above)
Comment (grey-kristy) [reply cm/job.py]: yes, but I think it is more readable

File: cm/job.py (hunk 2 above)
Comment (a-alferov) [reply cm/job.py]: Ok.

File: cm/job.py (hunk 1 above)
Comment (a-alferov) [reply cm/job.py]: Ок

**Pull request merged**